requirements.txt

```diff
@@ -70,6 +70,7 @@ multiformats==0.3.1.post4
 multiformats-config==0.3.1
 oauthlib==3.2.2
 packaging==24.2
+Pillow==11.0.0
```

Comment:
thanks! this file is for the https://granary.io/ webapp and API; you'll also need to add Pillow to `setup.py`. ideally without a min version, or with as loose a version req't as possible, but I don't know exactly when `Image.open` and `.size` were first added.

Reply:
I would guess they've been there since the start!

Reply:
Ah yeah it's a fork of PIL, so we're good

Reply:
Done!